src/ring_theory/localization/basic.lean

```diff
@@ -868,6 +875,15 @@ by rw [mk_eq_monoid_of_mk'_apply, mk', to_localization_map_eq_monoid_of]
 @[simp] lemma mk_eq_mk' : (mk : R → M → localization M) = is_localization.mk' (localization M) :=
 mk_eq_monoid_of_mk'
 
+lemma mk_algebra_map {A : Type*} [comm_semiring A] [algebra A R] (m : A) :
+  (mk (algebra_map A R m) 1 : localization M) = algebra_map _ _ m :=
```

Comment:
This would be easier to read if you fill out these underscores. Is this right?
```suggestion
  mk (algebra_map A R m) 1 = algebra_map A(localization M) m :=
```

Reply:
Yes it is correct